docs/diagrams/ContactClassDiagram.puml

```diff
@@ -0,0 +1,26 @@
+@startuml ContactClassDiagram
+
+
+    class Contact {
+    }
+
+    class Name
+    class Phone
+    class Email
+    class Address
+    class GitHubUsername
+    class TechStack
+    class Tag
+    class ProfilePicture
+
+    Contact *-- Name
+    Contact *-- Phone
+    Contact *-- Email
+    Contact *-- Address
+    Contact *-- GitHubUsername
+    Contact *-- TechStack
```

Comment:
A contact can have multiple tags and techstacks, so it should be like this? ```Contact *--> "*" Tag``` and ```Contact *--> "*" TechStack```

Reply:
good catch!